modules/ardrivo/test/String.cxx

```diff
@@ -0,0 +1,227 @@
+#include <catch2/catch.hpp>
+#include "WString.h"
+
+using Catch::Contains;
+using Catch::EndsWith; 
+
+TEST_CASE("Test compare a string to another" "[compareTo]")
+{
+	String test = "test";
+	String test2 = "test";
+	String test3 = "test";
+	String test4 = "est";	
+	REQUIRE(!test.compareTo(test2));
+	REQUIRE(test3.compareTo(test4));
```

Comment:
`String::compareTo` is a three-way comparison; you are only testing for equality/inequality; insufficient test